README.md

```diff
@@ -129,6 +134,19 @@ And on my Windows 10 machine:
 └──────────────────────────┘
 ```
 
+macOS: 
```

Comment:
Could be interesting to show what version of OSX you have, WDYT?

Reply:
I had `macOS 10.15.6` first, then changed it to `macOS Catalina` and finally I thought: "Well, it is just an example and Mac users will see that it is a Homebrew environment - the version doesn't matter". But I can add it for sure I you prefer that.

Reply:
Newcomers might not even be aware of Homebrew, so... I think all information are useful 😉

Reply:
I like your positive attitude! I couldn't imagine that a newcomer would stumble upon this tool and understands its usage before finding Homebrew 😁 but why not?! I added the version in the last commit

Reply:
It's more than if this project grows and it becomes a good tool for newcomers, I wish they find it newcomer-friendly 😄